.goreleaser.yml

```diff
@@ -173,10 +173,10 @@ builds:
       - -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
 
 nfpms:
-  - vendor: 0xPolygon
-    homepage: https://polygon.technology
-    maintainer: Polygon Team <team@polygon.technology>
-    description: Polygon Blockchain
+  - vendor: 0xShibarium
+    homepage: https://shibarium.shib.io/
+    maintainer: Shibarium Team <enquiries@shib.io>
```

Comment:
i did also the same changes in template.. we need to check for email do we have these